workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/ActionNode.java

```diff
@@ -152,6 +153,18 @@ public void setProcessInfo(ProcessInfo processInfo) {
             containingNodes.add(lofActionNode);
             containingNodes.add(dataQualityActionNode);
             containingNodes.add(fileRegistrationNode);
+        } else if (processInfo.getProcessTypeId() == SEMANTIC_ACTION) {
+
+        } else if (processInfo.getProcessTypeId() == INGESTION) {
+
+        } else if (processInfo.getProcessTypeId() == EXPORT_ACTION) {
+
+        } else if (processInfo.getProcessTypeId() == IMPORT_ACTION) {
+
+        } else if (processInfo.getProcessTypeId() == DQ_PARENT_ACTION) {
+
+        } else if (processInfo.getProcessTypeId() == HIVE_GEN_PARENT_ACTION) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00108)